### PR TITLE
Fix vulnerabilities by new image

### PR DESCRIPTION
**Pull request by ChunxiAlexLuo**
https://github.com/open-cluster-management/backlog/issues/14441

https://quay.io/repository/open-cluster-management/grc-ui-api/manifest/sha256:3955e019ce0ee9948e90bea856ffed5f8cab6e5666bd95576a46e4d007e81ed2?tab=vulnerabilities

### CODE REVIEW

**Comment by ChunxiAlexLuo** [comment]
https://quay.io/repository/open-cluster-management/grc-ui-api/manifest/sha256:3955e019ce0ee9948e90bea856ffed5f8cab6e5666bd95576a46e4d007e81ed2?tab=vulnerabilities

File: Dockerfile [hunk 1]
```diff
@@ -13,10 +13,8 @@ RUN make install
 RUN make build-prod
 RUN make prune
 
-FROM registry.access.redhat.com/ubi8/nodejs-14:1
-USER root
-RUN yum -y remove nodejs-nodemon
-RUN yum -y --nobest update
+FROM registry.ci.openshift.org/open-cluster-management/common-nodejs-parent:nodejs-14
+RUN microdnf update
```
Comment (dhaiducek) [inline Dockerfile:17]: I think we need to add:
```
RUN ./install-npm.sh
```
https://github.com/open-cluster-management/common-nodejs-parent/blob/6b20341a3d71efdb5e541a2b2b2727a6dcfb71c7/Dockerfile#L11-L14

Comment (dhaiducek) [reply Dockerfile:17]: Oh, nevermind--we run `node` directly.

Comment (dhaiducek) [reply Dockerfile:17]: Sorry for the back-and-forth between...myself. 😆 

Do we need to update the builder image at the top of the file? If so, we'll need to install NPM.

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,7 @@
 # Copyright Contributors to the Open Cluster Management project
 
-FROM registry.access.redhat.com/ubi8/nodejs-14:1 as builder
-
-USER root
-RUN yum install git -y
+FROM registry.ci.openshift.org/open-cluster-management/common-nodejs-parent:nodejs-14
+RUN microdnf install git -y
```
Comment (dhaiducek) [inline Dockerfile:4]: The `make` targets below this rely on NPM, so I think we'll need to add:
```
RUN ./install-npm.sh
```
https://github.com/open-cluster-management/common-nodejs-parent/blob/6b20341a3d71efdb5e541a2b2b2727a6dcfb71c7/Dockerfile#L11-L14

**Review by dhaiducek — CHANGES_REQUESTED** [review]

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**